config/prometheus/rbac-lvm-op-metric-role-n-rolebindings.yaml

```diff
@@ -0,0 +1,33 @@
+# Reference link: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: lvm-operator-metrics
+  namespace: system
```

Comment:
- do not add namespace here, it'll be inferred from `config/default/kustomization.yaml`, same for all remaining manifests
- or are you targeting ns other than lvm operator namespace?

Reply:
the file is removed nowe